bsc.verilog/derived_bits/derived_bits.exp

```diff
@@ -42,3 +42,12 @@ compare_verilog c0Test.v
 compare_verilog c1Test.v
 
 compare_verilog origTest.v
+
+compile_verilog_pass DerivedEnums.bsv
+
+# Test that pack . unpack optimizes away for enums (including non-contiguous)
+compile_verilog_pass DerivedEnums.bsv
+
+compare_verilog type1Reg.v
```

Comment:
There are six synthesized modules and you're only testing three of them here.  I assume that you also intended to have these lines in the exp-file:
```
compare_verilog type1Test.v
compare_verilog type2Test.v
compare_verilog type3Test.v
```

Reply:
Also fixed